tests/tests.rs

```diff
@@ -47,3 +50,46 @@ async fn fairing_init_with_specific_traces_sampler() {
 
     assert!(hub.client().is_some());
 }
+
+async fn init_rocket_using_figment(figment: Figment) {
+    rocket::custom(figment)
+        .attach(RocketSentry::fairing())
+        .ignite()
+        .await
+        .expect("Rocket failed to ignite");
+}
+
+fn sentry_current_hub_environment() -> String {
+    let client = Hub::current().client().unwrap();
```

Comment:
Hrm, I realize now that these tests are based on mutating and checking global state as well.

Although running a few hundred iterations of `cargo test`, it hasn't failed yet.

If you're up for it, let's address this in a follow-up PR by using a custom `Hub` instance. https://github.com/intgr/rocket-sentry/pull/80#discussion_r1978085424